spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReadProjection.java

```diff
@@ -110,7 +110,7 @@ public static void stopSpark() {
   @Override
   protected Record writeAndRead(String desc, Schema writeSchema, Schema readSchema, Record record)
       throws IOException {
-    File parent = temp.newFolder(desc);
+    File parent = temp.resolve(desc).toFile();
```

Comment:
given that you're adjusting this file, shouldn't this also be updated to use JUnit5? There is `@BeforeClass` / `@AfterClass` usage. For parameterizing the test you can use the annotations introduced by https://github.com/apache/iceberg/pull/9161

Reply:
I wanted to create a PR with only non-paramterized changes. This file was using the tempDir from its base class, which was now a JUnit5 implementation. So I only made this small change. The changes with parameterizing annotations would be in another PR. 
Or is it ok to do it all in 1 PR? That might make the PR hard to review

Reply:
I just want to make sure that we don't partially update files and then forgot about updating them fully. I'm ok doing it in an immediate follow-up PR